docs/sql-data-sources-orc.md

```diff
@@ -241,7 +241,7 @@ Data source options of ORC can be set via:
   <tr>
     <td><code>compression</code></td>
     <td><code>zstd</code></td>
-    <td>compression codec to use when saving to file. This can be one of the known case-insensitive shorten names (none, snappy, zlib, lzo, zstd and lz4). This will override <code>orc.compress</code> and <code>spark.sql.orc.compression.codec</code>.</td>
+    <td>compression codec to use when saving to file. This can be one of the known case-insensitive shorten names (none, snappy, zlib, lzo, zstd, lz4 and brotli). This will override <code>orc.compress</code> and <code>spark.sql.orc.compression.codec</code>.</td>
```

Comment:
Like Parquet, we need a `Note` here. 

https://github.com/apache/spark/blob/5f48931fcdf712c3646a368ae9643b5ccba00db2/docs/sql-data-sources-parquet.md?plain=1#L488

Reply:
Thanks. Updated.